src/include/ipc/stream.h

```diff
@@ -56,6 +56,7 @@ enum sof_ipc_frame {
 	SOF_IPC_FRAME_FLOAT,
 	/* other formats here */
 	SOF_IPC_FRAME_S24_3LE,
+	SOF_IPC_FRAME_U8,
```

Comment:
This needs an ABI bump as this is adding an item to the IPC3 interface. I.e. just bump the MINOR version in this PR.

Reply:
ack, a minor update is all :)